src/routes/FullPageModal/FullPageModal.scss

```diff
@@ -30,7 +30,7 @@ $size-sidebar-width: 13rem;
   padding-right: 32px;
   .left-sidebar-fixed {
     position: fixed;
-    padding-top: 220px;
+    padding-top: 160px;
```

Comment:
why this change?

Reply:
Because I added another listing in the left side, the bottom of the list was getting too low in smaller screens.